### PR TITLE
Rohit supabase

### PR DESCRIPTION
**Pull request by Rohit-k-singh16**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,7 +94,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "GENERATE_SOURCEMAP=false react-scripts build && npm run copy-build && npm run create-check-sum && cap sync && npm run build-storybook",
+    "build": "set GENERATE_SOURCEMAP=false && react-scripts build && npm run copy-build && npm run create-check-sum && cap sync",
```
Comment (bhanugummana) [inline package.json:97]: revert this changes

File: android/app/src/main/assets/capacitor.config.json [hunk 2]
```diff
@@ -21,6 +21,14 @@
 		"Keyboard": {},
 		"LocalNotifications": {
 			"smallIcon": "chimple_monkey_icon"
+		},
+		"GoogleAuth": {
+			"scopes": [
+				"profile",
+				"email"
+			],
+			"androidClientId": "1034970790912-jj83tjb3s5t27554iuhhtoe3etou7tfs.apps.googleusercontent.com",
```
Comment (bhanugummana) [inline android/app/src/main/assets/capacitor.config.json:30]: Revert this changes

File: src/components/displaySubjects/SelectIconImage.tsx [hunk 3]
```diff
@@ -51,12 +51,20 @@ const SelectIconImage: FC<{
       setIsLoading(false); 
     };
 
+
+    loadImages();
```
Comment (bhanugummana) [inline src/components/displaySubjects/SelectIconImage.tsx:55]: why are you calling loadImages twice??

File: src/components/displaySubjects/SelectIconImage.tsx [hunk 4]
```diff
@@ -51,12 +51,20 @@ const SelectIconImage: FC<{
...
     loadImages();
+
   }, [localSrc, webSrc, defaultSrc]);
 
   return (
     <div style={{ position: "relative", width: imageWidth, height: imageHeight }}>
+
+      {isLoading && ( 
+
       {isLoading && (
```
Comment (bhanugummana) [inline src/components/displaySubjects/SelectIconImage.tsx:66]: why is this twice??

**Review by bhanugummana — CHANGES_REQUESTED** [review]

File: src/components/displaySubjects/SelectIconImage.css [hunk 5]
```diff
@@ -1,4 +1,24 @@
...
+  width: calc(100% + 10%);
+  height: calc(100% + 10%);
+  opacity: 0;
+  transition: opacity 0.5s;
+}
+
+.imageLoaded {
+  opacity: 1;
+}
+=======
```
Comment (kvskandakumar) [inline src/components/displaySubjects/SelectIconImage.css:21]: remove this line it will break the app

**Review by kvskandakumar — CHANGES_REQUESTED** [review]
